tests/components/FieldValueDirectAnswer.test.tsx

```diff
@@ -13,16 +13,22 @@ import React from 'react';
 
 const fieldValueDAResult = fieldValueDAState.result as FieldValueDirectAnswerType;
 
+
+
 describe('FieldValue direct answer', () => {
 
-  it('executes viewDetailsClickHandler when click on "View Details" link', () => {
+  it('executes viewDetailsClickHandler when click on "View Details" link', async () => {
     const viewDetailsClickHandler = jest.fn();
     render(<FieldValueDirectAnswer
       result={fieldValueDAResult}
       viewDetailsClickHandler={viewDetailsClickHandler}/>
     );
-    userEvent.click(screen.getByRole('link', { name: 'View Details' }));
-    expect(viewDetailsClickHandler).toHaveBeenCalledTimes(1);
+    const viewDetailsLink = screen.getByRole('link', { name: 'View Details' });
+    await userEvent.click(viewDetailsLink);
+
+    waitFor(() => {
```

Comment:
now that we're awaiting all the user events, can we remove all the waitFor's in our tests?

Reply:
done.